cmd/variables.go

```diff
@@ -86,7 +92,7 @@ func (h *Handler) VariablesSet(ctx context.Context, req *entity.CommandRequest)
 	fmt.Print(ui.Heading(fmt.Sprintf("Updated %s for \"%s\"", strings.Join(updatedEnvNames, ", "), environment.Name)))
 	fmt.Print(ui.KeyValues(*variables))
 
-	err = h.redeployAfterVariablesChange(ctx, environment)
+	err = h.redeployAfterVariablesChange(ctx, environment, serviceID, noRedeploy)
 	if err != nil {
```

Comment:
Ideally the redeployAfterVariablesChange shouldn't be aware of the flag. Should just have an `if !noRedeploys` at the top level here and keep the function signature clean